docs/src/pages/[platform]/getting-started/usage/usage.angular.mdx

```diff
@@ -1,6 +1,6 @@
 import { Alert, Text } from '@aws-amplify/ui-react';
 
-<Alert variation="info" marginBottom="1rem">
```

Comment:
We don't need these bottom margins anymore now that the site has a vertical rhythm in place.